clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/StreamsAvroExample.java

```diff
@@ -69,15 +68,12 @@ public static void main(String[] args) throws Exception {
         props.put(StreamsConfig.APPLICATION_ID_CONFIG, "demo-streams-avro-1");
         // Disable caching to print the aggregation value after each record
         props.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
-        props.put(StreamsConfig.REPLICATION_FACTOR_CONFIG, 3);
         props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
 
         final Serde<DataRecordAvro> dataRecordAvroSerde = new SpecificAvroSerde<>();
         final boolean isKeySerde = false;
-        Map<String, Object> SRconfig = new HashMap<>();
-        SRconfig.put(AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, props.getProperty(AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG));
-        SRconfig.put("schema.registry.basic.auth.user.info", props.getProperty("schema.registry.basic.auth.user.info"));
-        SRconfig.put("basic.auth.credentials.source", props.getProperty("basic.auth.credentials.source"));
+        // Read Schema Registry URL, and optionally authentication configuration, from the props
```

Comment:
I'd drop this comment line as it's no longer applicable as we're reading the auth configuration from the file

Reply:
Dropped it like it's hot.